exec/pom.xml

```diff
@@ -198,6 +204,7 @@
                                 <argument>com.optum.sourcehawk.exec.ScanCommand</argument>
                                 <argument>com.optum.sourcehawk.exec.ValidateConfigCommand</argument>
                                 <argument>com.optum.sourcehawk.exec.FixCommand</argument>
+                                <argument>com.optum.sourcehawk.exec.FlattenCommand</argument>
```

Comment:
Please also add a file exists enforcer lower in this `pom.xml` to make sure the `manpage` is generated.

Reply:
done